lib/commands/command-update-nodebuild.bash

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+
+set -eu -o pipefail
+
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../utils.sh"
+
+: "${ASDF_NODEJS_NODEBUILD_HOME=$ASDF_NODEJS_PLUGIN_DIR/.node-build}"
+: "${ASDF_NODEJS_NODEBUILD_REPOSITORY=https://github.com/nodenv/node-build.git}"
+
+ensure_updated_project() {
+  local pull_exit_code=0 output=
+
+  if ! [ -d "$ASDF_NODEJS_NODEBUILD_HOME" ]; then
+    printf "Cloning node-build...\n"
+    git clone "$ASDF_NODEJS_NODEBUILD_REPOSITORY" "$ASDF_NODEJS_NODEBUILD_HOME"
```

Comment:
In the other version managers (Ruby and Erlang come to mind) we lock the builder to a specific tag, and only bump it manually when it makes sense. Do you think that locking node-build to a specific tag will be problematic?

Reply:
What would be the benefits of pinning the version of `node-build`? The point that I can see is that we review the changes upstream before pushing them to users.

The advantage of always having `node-build` updated is that we don't need to create PRs on our side for newer versions

Reply:
Is an update of `node-build` required in order to build new versions? For example, say nodejs team tagged a new release, and a user tried to install it via this plugin immediately. Would downloading a new version of `node-build` need to occur first?

Reply:
> Is an update of `node-build` required in order to build new versions? For example, say nodejs team tagged a new release, and a user tried to install it via this plugin immediately. Would downloading a new version of `node-build` need to occur first?

Yes, `node-build` needs the definition of the new version on their repository